src/main/resources/db/migration/V14__add_fingerpint_to_tables.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+ALTER table offender
+    Add column fingerprint BOOLEAN NULL;
+
+ALTER table defendant
+    Add column fingerprint BOOLEAN NULL;
+
+ALTER table prisoner
+    Add column fingerprint BOOLEAN NULL;
```

Comment:
I think the default value should be false

Reply:
Done 👍